arbitrum-docs/node-running/how-tos/running-an-orbit-node.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: 'How to run an orbit node'
+description: Learn how to run an Arbitrum orbit node on your local machine.
+sme: Mehdi Salehi
+sidebar_label: Run an orbit node
+sidebar_position: 12
+content-type: how-to
+---
+
+This guide provides a step-by-step process for running an Orbit node on your local machine. To initiate an Orbit node, the `"chain info"` json string of the orbit chain, which includes crucial configurations required for the Orbit chain's operation. The `"chain info"` json string should be provided by the orbit chain owner, and the node runner can consume it to run the node for that specific orbit chain. An example of the "chain info" json string is available in this section. 
```

Comment:
```suggestion
This guide provides a step-by-step process for running a node for an Orbit chain on your local machine. To initiate an Orbit node, the `"chain info"` json string of the Orbit chain, which includes crucial configurations required for the Orbit chain's operation. The `"chain info"` json string should be provided by the Orbit chain owner, and the node runner can consume it to run the node for that specific Orbit chain. An example of the "chain info" json string is available in this section. 
```

I think we should call them "nodes for an Orbit chain" instead of "Orbit node", because the reader might think that it's a different type of node (than a full node).

In "in this section", can you include a link to that section?